virtcontainers/pkg/cgroups/manager.go

```diff
@@ -176,7 +163,7 @@ func New(config *Config) (*Manager, error) {
 }
 
 // read all the pids in cgroupPath
-func readPids(cgroupPath string) ([]int, error) {
+func (m *Manager) readPids(cgroupPath string) ([]int, error) {
```

Comment:
does this change is really needed? I don't think so

Reply:
No, it is not necessary. I'll move it back to being a function.